<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trainer_win</class>
 <widget class="QWidget" name="Trainer_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainer_win</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>931</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="info_tab">
    <property name="styleSheet">
     <string notr="true">#info_tab{border-image: url(C:/Users/youssef albali/OneDrive/Desktop/SEM4/CP4/0f2c9051-gr12-repo/Project/GymAppQt/purple_background.png)}</string>
    </property>
    <attribute name="title">
     <string>Info</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>611</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #EBE9F5;
border-radius: 25px;
</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>291</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FFFFFF;
border-radius: 25px;
</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7D43E2</string>
       </property>
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7D43E2</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>58</x>
         <y>140</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #7D43E2</string>
       </property>
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
      <widget class="QLabel" name="category_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="surname_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Trainer Info</string>
       </property>
      </widget>
     </widget>
     <widget class="QToolButton" name="change_password_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Change password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resign_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Resign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset ; 
border-radius: 11px;
border-color:  #7D43E2;
border-width: 1px;
background-color: #7D43E2;
color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="schedule_tab">
    <attribute name="title">
     <string>schedule</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>801</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

background-color: #FFFFFF;
selection-background-color: #5492dc;
border: 1px solid;
border-color: #f3f3f3;
color: #080c2b;
border-radius:15px;
padding: 30px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Member</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Session </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>About</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sessions</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
